eclair-core/src/main/scala/fr/acinq/eclair/io/PeerConnection.scala

```diff
@@ -207,8 +207,9 @@ class PeerConnection(keyPair: KeyPair, conf: PeerConnection.Conf, switchboard: A
             log.debug(s"received pong with latency=$latency")
             cancelTimer(PingTimeout.toString())
           // we don't need to call scheduleNextPing here, the next ping was already scheduled when we received that pong
-          case None =>
+          case _ =>
             log.debug(s"received unexpected pong with size=${data.length}")
+            d.transport ! Warning(s"invalid pong length (${data.length})")
```

Comment:
Nit² (it's not necessarily an issue with the length, it could be  an unexpected pong)

```suggestion
            log.debug(s"received unexpected pong with length=${data.length}")
            d.transport ! Warning(s"invalid pong with length=${data.length}")
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1863/commits/3c7c0eed9489e471062615ff40bd3a966f5a5857